### ISSUE TITLE
SDAPI Collections Instructions

### ISSUE THREAD

**Issue by ShanaLMoore** [scientist-softserv/adventist-dl#402]
# Story

Please review for extra context: [slack convo/thread](https://assaydepot.slack.com/archives/C0313NJV9PE/p1683555308399759).

`"Hey team, I realized something that affects the launch of SDAPI. To facilitate some work on the ADL tenant, we turned off the relationships that deposit works into their containing collections. Now we need to work on loading SDAPI, and those relationships are less of a roadblock for this tenant. Can we turn that feature back on while we work to finish the SDAPI migration?" - Katharine`

Note: Work has been done for ADL to turn off the creation of collections when importing an OAI feed. Some form of this will likely need to be undone/changed, to accommodate SDAPI collections.
- related PR: https://github.com/scientist-softserv/adventist-dl/pull/285 

Note: SDAPI stores its collections in a set, per Rob. Consider ingesting the set first to pre-create the collections. Previously imported OAI works may need to be re-ingested. We could also consider identifying those records, turning them into a CSV to then be uploaded into Bulkrax again, to establish the relationship with the collection.

 

# Acceptance Criteria

- [ ] OAI ingested works saves to collections. 

# Testing Instructions and Sample Files

- TBD

# Notes

I am not seeing a sdapi::collection set option in the bulkrax dropdown. Ask Rob to clarify.

<img width="740" alt="image" src="https://github.com/scientist-softserv/adventist-dl/assets/10081604/8a8ea74f-e9a0-4e11-aa9c-032e96a01f7b">

**Comment by kirkkwang** [comment]
After looking into this with Rob, it looks like the `adl:periodical` is what he remembered to have collections at some point.  SDAPI doesn't look like it has a collection set

**Comment by kirkkwang** [comment]
I've tested this workflow:

1. Ingest all SDAPI records
2. Remove all collections if there are any left over from old data
3. Katharine creates a CSV with all the collections with unique identifiers and imports those
4. Katharine creates a huge CSV with all the records that have relationships and imports to update the relationships

The CSV of the collection example:

identifier | model | title | description
-- | -- | -- | --
test-collection-1 | Collection | Adventist Review | This is a test collection
test-collection-2 | Collection | Southwestern Union Record | This is a test collection
test-collection-3 | Collection | Lake Union Herald | This is a test collection
test-collection-4 | Collection | Record (South Pacific Division) | This is a test collection

***(identifiers can be whatever, but needs to be unique)***
***(add metadata as needed)***

The CSV to update the works example:

identifier | model | title | parents
-- | -- | -- | --
13128990 | JournalArticle | $1 million for Reading Hospital | test-collection-1
13128988 | JournalArticle | $0.13 postage stamp convert | test-collection-2
13128992 | JournalArticle | $1 million in tithe reached by one church | test-collection-3
13128989 | JournalArticle | $1 million Bible ministry makes major impact | test-collection-1
13128991 | JournalArticle | $1 million grant | test-collection-4

Those four headers (`identifier`, `model`, `title`, and `parents`) are required to make the relationships work.

**Comment by ShanaLMoore** [comment]
cc @KatharineV See Jeremy's [recommendation](https://github.com/scientist-softserv/adventist-dl/issues/402#issuecomment-1549917862) from above. We noticed that a sdapi::collection set does not exist, and when looking closer at the data it sounded very similar to the collection problems we saw in the non-SDAPI


 Kirk has tested it and it's a viable workaround. Let us know if you have any questions.
